### PR TITLE
KAA-1166: Configuration management - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Key-platform-features/Configuration-management/index.md [hunk 1]
```diff
@@ -10,88 +10,87 @@ sort_idx: 60
...
-Kaa developer can provision the configuration schema using Administration UI (see image below) or [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Configuration/saveConfigurationSchema).
+## Updating configuration data
+
+To provision a configuration schema:
+
+- Use the [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Configuration/saveConfigurationSchema).
+
+OR
+
+- Use the [Administration UI]({{root_url}}Glossary/#administration-ui) to open the **Configuration** page under the **Schemas** section of your application and click **Add schema**.
```
Comment (ikulikov) [inline doc/Programming-guide/Key-platform-features/Configuration-management/index.md:27]: Procedure of adding configuration schema is outdated. This part should be updated according to new procedure based on CTL.

File: doc/Programming-guide/Key-platform-features/Configuration-management/index.md [hunk 2]
```diff
@@ -10,88 +10,87 @@ sort_idx: 60
...
+For more information about schemas in Kaa, see [Common type library]({{root_url}}Programming-guide/Key-platform-features/Common-Type-Library).
 
-* Automatic generation of the configuration object model in the endpoint SDK
-* Automatic generation of the default configuration
-* Enforcement of the configuration integrity
-* Endpoint-specific configuration view that is based on the endpoint membership in endpoint groups
+Since Kaa operates on uniformly structured data, you can use the following features:
 
-# Configuration management
+* Automatic generation of the configuration object model in the endpoint SDK.
```
Comment (Kirill380) [inline doc/Programming-guide/Key-platform-features/Configuration-management/index.md]: @ashvayka what does this sentence mean? Look very complicated

File: doc/Programming-guide/Key-platform-features/Configuration-management/index.md (hunk 1 above)
Comment (Kirill380) [reply doc/Programming-guide/Key-platform-features/Configuration-management/index.md:27]: Better do like on Notification page -- provide link to how setting client-side EP profile schema

Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Configuration-management/index.md:27]: Fixed.

File: doc/Programming-guide/Key-platform-features/Configuration-management/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Configuration-management/index.md]: I rephrased the whole paragraph. Should be clearer now.

**Pull request merged**